frontend/.env.docker

```diff
@@ -0,0 +1,2 @@
+REACT_APP_API_URL=http://localhost:8080
+REACT_APP_GITHUB_COMMUNITY_URL=https://api.github.com/repos/sladyn98/custom-distribution-service-community-configurations/contents/configurations
```

Comment:
I was able to change the value of REACT_APP_GITHUB_COMMUNITY_URL to:
```
REACT_APP_GITHUB_COMMUNITY_URL=https://api.github.com/repos/martinda/custom-jenkins-distro-configs-ibrary/contents/configurations
```
and it works. My configurations though report a 404 in the editor window.

Reply:
That is strange, it should not happen like that I will look into it

Reply:
This works fine when I change it, maybe could be a network error